addons/website/static/src/snippets/s_dynamic_snippet/000.scss

```diff
@@ -1,4 +1,7 @@
 .s_dynamic {
+    &.o_dynamic_empty {
+        display: none;
+    }
```

Comment:
Would not this be ignored if the "hide in mobile" option is enabled as d-md-block uses `!important` ?

Reply:
:facepalm: I actually thought about this, but then I could not spot the difference.
Re-testing now at a cooler temperature I actually saw it. Sorry...